bodge/TF_D3D4_tb_writer.vhd.bodge

```diff
@@ -0,0 +1,15 @@
+  -- A bodge for TrackBuilder to write TF concatenated track+stub data for L1L2.
+  -- (Needed to compare with emData/).
+  writeTF_D3D4_297 : entity work.FileWriterFIFO
+  generic map (
+    FILE_NAME  => FILE_OUT_TF&memory_enum_to_string(D3D4)&outputFileNameEnding,
+    FIFO_WIDTH  => 297
+  )
+  port map (
+    CLK => CLK,
+    DONE => FT_DONE,
+    WRITE_EN => TW_D3D4_stream_A_write,
```

Comment:
@aryd Since the TrackBuilder can output invalid tracks, you should use the valid bit of the track word to enable writing:
```suggestion
    WRITE_EN => (TW_D3D4_stream_A_write and TW_D3D4_stream_AV_din(103)),
```

Reply:
Fixed.